Lord Forte Backlog.rb

```diff
@@ -181,7 +181,6 @@ def initialize
     #self.active = false
      self.active = true
     self.openness = 0
-    self.padding *= 2

```

Comment:
Removing the excess padding frees up space to add a gap between the face and the text.